src/components/Card/CardList.tsx

```diff
@@ -78,6 +82,7 @@ export function CardList({
     updateCurrentIndex(index - 1);
     if (direction === "left" || direction === "down") {
       removeCard(cards[index]);
+      setSkipped(true);
```

Comment:
It seems like here, we're setting `skipped` to `true` for all `cards`, so that when we use it later we're checking if any `cards` were `skipped`. I would've thought that the management of `skipped` would be done similarly to:
```
  const getCompletedContentLinks = (): string[] => {
    return JSON.parse(localStorage.getItem("eras.completedCardLinks") || "[]");
  };
```

but as this is the approach you've taken then I will defer to your thinking. There is no material difference at this point but it would move us in the direction of managing a user's relationship with individual cards.

Reply:
This is not about which cards were skipped, but whether any cards were skipped. I will rename!